src/arith/int_set.cc

```diff
@@ -730,12 +765,30 @@ Map<Var, IntSet> ConvertDomMap(
   return dmap;
 }
 
-IntSet EvalSet(PrimExpr e,
-               const Map<Var, IntSet>& dom_map) {
+IntSet EvalSet(PrimExpr e, const Map<Var, IntSet>& dom_map,
+               const std::unordered_map<IterVar, Range>& rmap) {
   Analyzer ana;
+  // Bind ana with rmap
+  for (auto entry : rmap) {
+    ana.Bind(entry.first->var, entry.second);
```

Comment:
Let us try to upgrade the API away from the EvalSet, and directly use https://github.com/apache/incubator-tvm/blob/master/include/tvm/arith/analyzer.h#L366

Reply:
Don't need this change any more.